### PR TITLE
Adds clarification about multi-cluster configs for 2.0

### PR DESCRIPTION
**Pull request by bstuder99**

### CODE REVIEW

**Comment by thomasvn** [comment]
I'd adjust the wording maybe? They are not 100% blocked. However currently 2.0 would require them set up a bucket. Details in https://github.com/kubecost/docs/blob/main/install-and-configure/install/multi-cluster/federated-etl/thanos-migration-guide.md

**Comment by bstuder99** [comment]
Do you specifically mean the object store the user creates in Step 1 of that article?

**Comment by thomasvn** [comment]
@bstuder99 Correct!

**Comment by bstuder99** [comment]
@thomasvn Slipped my mind for a few days but I've updated it accordingly

File: install-and-configure/install/multi-cluster/federated-etl/federated-etl.md [hunk 1]
```diff
@@ -6,6 +6,12 @@ Federated ETL is only supported for Kubecost Enterprise plans.
 
 Federated extract, transform, load (ETL) is Kubecost's method to aggregate all cluster information back to a single display described in our [Multi-Cluster](/install-and-configure/install/multi-cluster/multi-cluster.md#enterprise-federation) doc. Federated ETL gives teams the benefit of combining multiple Kubecost installations into one view.
 
+There are several primary advantages for using ETL federation:
+
+1. For environments that already have a Prometheus instance, Kubecost only requires a single pod per monitored cluster
+2. Many solutions that aggregate Prometheus metrics (like Thanos), are often expensive to scale in large environments
+3. With [Aggregator](/install-and-configure/install/multi-cluster/federated-etl/aggregator.md) as of Kubecost v2.0, ETL federation is more scalable.
+
```
Comment (thomasvn) [inline install-and-configure/install/multi-cluster/federated-etl/federated-etl.md:14]: I believe I removed this text in a different PR?

**Comment by thomasvn** [comment]
I'm still hesitant about the wording "temporarily blocked". For reference, this is the error message we return to users upon attempting a `helm upgrade`. https://github.com/kubecost/cost-analyzer-helm-chart/blob/8ce0106b2825ebabaacf3e43f957d80fa381592c/cost-analyzer/templates/_helpers.tpl#L59. Maybe we could do something more in line with that?

**Comment by bstuder99** [comment]
It sounds like it conveys that same idea. It's "temporarily blocked" until Kubecost rolls out an update that supports multi cluster configs.

**Comment by thomasvn** [comment]
Ok! If you feel those two texts are similar then I'm good with it.

File: install-and-configure/install/multi-cluster/federated-etl/federated-etl.md (hunk 1 above)
Comment (bstuder99) [reply install-and-configure/install/multi-cluster/federated-etl/federated-etl.md:14]: Possibly got readded during a merge conflict. Good catch, I'll remove it.

**Review by thomasvn — APPROVED** [review]

**Pull request merged**